### PR TITLE
Add demo tips

### PR DESCRIPTION
**Pull request by karenword**
This can & should be linked from the handbook demo section, but I decided that Instructor Notes was the best place to collect this, as we can continue to collect initialed "I like to x" style contributions as opposed to codifying things (as implied by handbook additions).

### CODE REVIEW

**Comment by ChristinaLK** [comment]
I think this is great. :)

**Comment by ChristinaLK** [comment]
One idea: add a link to 
https://docs.carpentries.org/topic_folders/instructor_training/trainers_guide.html#running-a-teaching-demonstration
?

**Comment by karenword** [comment]
Yes! Done :)

File: _extras/guide.md [hunk 1]
```diff
@@ -247,6 +247,28 @@ group is interested.
...
 
+## VI. Teaching Demonstration Tips
+
+Suggested points to make during introductions:
+* This is not a high-stakes test! This is a friendly opportunity to give and receive feedback. On occasion, we do invite 
+people to return for a second try. If we do this, your follow-up email will contain very clear instructions on exactly what
+we'd like you to change on your next visit. This does not mean we don't want you as an instructor, it means we want you to be
+ready to put your best foot forward in the classroom and we are here to help you do that. (KW)
+* I will not tell anyone that they have passed or ask anyone to repeat during this session. That information will come in the follow up email. (KW)
+* The feedback you give sometimes says as much or more about your approach to instruction as the demo itself. (KW)
```
Comment (lexnederbragt) [inline _extras/guide.md]: Do you mean 'the feedback you give on your own demo' or on the demo of others?

File: _extras/guide.md [hunk 2]
```diff
@@ -247,6 +247,28 @@ group is interested.
...
+
+Different ways of managing feedback during the demo:
+* Create your own etherpad and populate it with episode links, feedback templates, whatever. 
+* Use +/- content delivery rubric. (Or use "improvement" in place of "-" as suggested by MC). Keep in mind that some learners
```
Comment (lexnederbragt) [inline _extras/guide.md]: Do you mean the '2x2' grid we use during training? Asking for feedback on delivery makes much sense, but I always feel asking for feedback on content feels like asking for feedback on the lesson material, which is beyond the control of the trainee...

File: _extras/guide.md [hunk 3]
```diff
@@ -247,6 +247,28 @@ group is interested.
...
+(like some Trainers) do not have the content expertise to be able to fully evaluate that category for all lessons.
+* If there is time (4 people or less) you can do feedback entirely verbally, or have them issue verbal feedback after writing 
+notes in the etherpad. If you do this, suggest having presenter give their own feedback, then other learners, then Trainer can
+summarize/disagree/add/emphasize as needed.
```
Comment (lexnederbragt) [inline _extras/guide.md]: Maybe add 'with less time, ask for written feedback from observers, and discuss that after which you add your own comments'

**Comment by lexnederbragt** [comment]
Darn, I had not refreshed the page in a while and didn't see it was already merged. Well, hopefully my comments are useful nonetheless :-)

**Comment by karenword** [comment]
Thanks @lexnederbragt ! We were hustling things through this weekend because of the release. The great thing about the instructor notes is that you can add your own perspective directly on there -- I can clarify what I meant for some of these, but where you have additions or caveats please feel free to add them in, along with your initials for credit!

**Pull request merged**